.github/workflows/dockerimage-latest.yml

```diff
@@ -17,15 +17,14 @@ jobs:
       env:
         HUB_NAME: ${{ secrets.DOCKER_NAME }}
         HUB_KEY: ${{ secrets.DOCKER_TOKEN }}
-      run: |
+        BLENDER_URL: https://download.blender.org/release/Blender3.2/blender-3.2.1-linux-x64.tar.xz
+        BLENDER_VERSION_STRING: "3.2"
+      run: |  
         docker login -u $HUB_NAME -p $HUB_KEY
-        
-        docker build . --pull --no-cache --file Dockerfile --tag crowdrender/blender-crowdrender:latest
-        
-        docker tag crowdrender/blender-crowdrender:latest crowdrender/blender-crowdrender:bl_3.2
-        
+        docker build . --pull --no-cache --file Dockerfile --tag crowdrender/blender-crowdrender:latest --build-arg BLENDER_BZ2_URL="$BLENDER_URL" BL_VERSION_SHORT="${BLENDER_VERSION_STRING}"
```

Comment:
why set `BLENDER_BZ2_URL`? 

Setting `BL_VERSION_SHORT` and `BL_VERSION_FULL` would make more sense, given that you assemble the url in the Dockerfile using those ARGs.

Reply:
you will get error here now because `BLENDER_VERSION_STRING` does not exist :V

Reply:
I've refactored a lot of this file now, could you take another look? The aim is to specify a set of versions to build and then push images based on those versions.